pep-0483.txt

```diff
@@ -51,23 +52,127 @@ Notational conventions
 Background
 ==========
 
-TODO: Explaint types as sets, subtyping, nominal and structural subtyping.
-Many examples.
+There are many definitions of the concept of type in the literature.
+Here we assume that type is a set of values and a set of functions that
+one can apply to these values. Note that such definition
+could have mathematical problems such as `Russel's paradox
+<https://en.wikipedia.org/wiki/Russell's_paradox>`_
+because the two above sets could be "too infinite".
+Nevertheless it is a good practical mental model for semantics of types.
```

Comment:
I think this is too academic to bother mentioning and I doubt that any of the readers care about this level of disclaimers.